lunabot_control/src/mpc_node.cpp

```diff
@@ -1,15 +1,29 @@
 #include <lunabot_control/mpc.h>
 
+bool traversal_enabled = true;
+
+void traversal_bool_callback(const std_msgs::Bool& msg) {
```

Comment:
What is this callback for? Is it just to check if MPC should actually publish or not?

Reply:
Yes

Reply:
Ideally it would stop MPC from generating paths too, so we don't waste compute power by making paths we don't need

Reply:
You can add this same flag to dstar node? There might be a cleaner solution but you can start with that for now.